extension/modules/discord/index.js

```diff
@@ -47,7 +47,7 @@ exporter(
       const res = MessageHeaderWithPronouns.MessageHeader.call(null, props)
       if (pronouns && showInChat) {
         res.props.children[1].props.children.push(
-          React.createElement('span', { style: { color: 'var(--text-muted)', fontSize: '.9rem' } }, ' • ', pronouns)
+          React.createElement('span', { style: { color: 'var(--text-muted)', fontSize: '.9rem', marginRight: '.9rem' } }, ' • ', pronouns)
```

Comment:
This may interfere with other injections done on other clients which do not expect the last element to have a margin. I believe the component receives `compact` as a prop, so you could conditionally apply this (`props.compact ? '.9rem' : 0`). Needs double checking though

Reply:
Reduced the sizing to .6rem so it fits better and added the restriction to compact mode. Hope that makes it a tad more compatible now